libs/zphysics/libs/JoltC/JoltPhysicsC.h

```diff
@@ -59,6 +59,15 @@ enum
     JPC_MAX_PHYSICS_BARRIERS = 8
 };
 
+typedef uint8_t JPC_PhysicsUpdateError;
```

Comment:
```suggestion
typedef uint8_t JPC_PhysicsUpdateResult;
```

Reply:
This is fine with me (that's what I would have named it too :) ), but just wanted to confirm given that the original enum is called Error:

external\zig-gamedev\libs\zphysics\libs\Jolt\Physics\EPhysicsUpdateError.h

/// Enum used by PhysicsSystem to report error conditions during the PhysicsSystem::Update call. This is a bit field, multiple errors can trigger in the same update.
enum class EPhysicsUpdateError : uint32
{
	None					= 0,			///< No errors
	ManifoldCacheFull		= 1 << 0,		///< The manifold cache is full, this means that the total number of contacts between bodies is too high. Some contacts were ignored. Increase inMaxContactConstraints in PhysicsSystem::Init.
	BodyPairCacheFull		= 1 << 1,		///< The body pair cache is full, this means that too many bodies contacted. Some contacts were ignored. Increase inMaxBodyPairs in PhysicsSystem::Init.
	ContactConstraintsFull	= 1 << 2,		///< The contact constraints buffer is full. Some contacts were ignored. Increase inMaxContactConstraints in PhysicsSystem::Init.
};